tdrs-backend/tdpservice/parsers/schema_defs/ssp/m1.py

```diff
@@ -14,92 +14,244 @@
           preparsing_validators=[
               validators.hasLength(150),
           ],
-          postparsing_validators=[],
+          postparsing_validators=[
+            validators.if_then_validator(
+                  condition_field='CASH_AMOUNT', condition_function=validators.isLargerThan(0),
+                  result_field='NBR_MONTHS', result_function=validators.isLargerThan(0),
+            ),
+            validators.if_then_validator(
+                  condition_field='CC_AMOUNT', condition_function=validators.isLargerThan(0),
+                  result_field='CHILDREN_COVERED', result_function=validators.isLargerThan(0),
+            ),
+            validators.if_then_validator(
+                  condition_field='CC_AMOUNT', condition_function=validators.isLargerThan(0),
+                  result_field='CC_NBR_MONTHS', result_function=validators.isLargerThan(0),
+            ),
+            validators.if_then_validator(
+                  condition_field='TRANSP_AMOUNT', condition_function=validators.isLargerThan(0),
+                  result_field='TRANSP_NBR_MONTHS', result_function=validators.isLargerThan(0),
+            ),
+            validators.if_then_validator(
```

Comment:
@jtimpe I missed this one. this check can be dropped because these data elements will be deprecated.